File: examples/node/basic_chat.js
import { Client } from "../../sdk/node/dist/index.js";

async function main() {
  const apiKey = process.env.KLEBERAI_API_KEY || "test-key";
  const baseUrl = process.env.KLEBERAI_BASE_URL || "http://localhost:18080";

  const client = new Client({ apiKey, baseUrl });

  console.log("Enviando pergunta...");
  try {
    const response = await client.chat("Explique o que é um LLM em poucas palavras.");
    console.log("\nResposta:");
    console.log(response.choices[0].message.content);
  } catch (error) {
    console.error("Erro:", error.message);
    if (error.body) {
      console.error("Detalhes:", JSON.stringify(error.body, null, 2));
    }
  }
}

main();

File: examples/python/basic_chat.py
import os
import sys

# Adiciona o diretório do SDK ao path para teste local
sys.path.append(os.path.join(os.path.dirname(__file__), "../../sdk/python"))

from kleberai import Client

def main():
    api_key = os.getenv("KLEBERAI_API_KEY", "test-key")
    base_url = os.getenv("KLEBERAI_BASE_URL", "http://localhost:18080")
    
    client = Client(api_key=api_key, base_url=base_url)
    
    print("Enviando pergunta...")
    try:
        response = client.chat("Explique o que é RAG em poucas palavras.")
        print("\nResposta:")
        print(response["choices"][0]["message"]["content"])
    except Exception as e:
        print(f"Erro: {e}")

if __name__ == "__main__":
    main()
